main/src/com/google/refine/exporters/CustomizableTabularExporterUtilities.java

```diff
@@ -114,27 +114,33 @@ static public void exportRows(
             }
         } else {
             int count = columnOptionArray.size();
-            
+
             columnNames = new ArrayList<String>(count);
             for (int i = 0; i < count; i++) {
                 JsonNode columnOptions = columnOptionArray.get(i);
                 if (columnOptions != null) {
                     String name = JSONUtilities.getString(columnOptions, "name", null);
                     if (name != null) {
                         columnNames.add(name);
-                        try {
-							columnNameToFormatter.put(name, ParsingUtilities.mapper.treeToValue(columnOptions, ColumnOptions.class));
-						} catch (JsonProcessingException e) {
-							e.printStackTrace();
-						}
+                        JsonNode reconSettingsNode = columnOptions.get("reconSettings");
+                        JsonNode dateSettingsNode = columnOptions.get("dateSettings");
```

Comment:
Instead of writing deserialization code by hand, I think it would be better to rely on Jackson deserialization with `ParsingUtilities.mapper.treeToValue(columnOptions, ColumnOptions.class)`. If this does not currently work well, there should be a way to fix the constructor of this class to make sure it deserializes things correctly.

Reply:
I found that `ParsingUtilities.mapper.treeToValue(columnOptions, ColumnOptions.class)` does not work well. It will lose the date format. The date format will always be as the default. But I do not know how to fix it, could you give some advice?
Writing deserialization code by hand can solve it, so I think the util may cause some bug.